Block/Adminhtml/Button/AbstractButton.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
```

Comment:
We can add already `declare(strict_types=1);` in classes, we started using it already, eg: https://github.com/SnowdogApps/magento2-menu/blob/develop/Block/NodeType/Product.php (we'll need to drop php 5.x compatibility en composer.json soon)

Reply:
Ok, added `declare(strict_types=1);`